documentation/selinux_fcontext.md

```diff
@@ -0,0 +1,48 @@
+[Back to resource list](../README.md#resources)
+
+# selinux_fcontext
+
+Set the SELinux context of files with `semanage fcontext`.
+
+## Actions
+
+| Action         | Description                                                                     |
```

Comment:
I'd say either change this to a list with `- :action - description` like the others or we make the others a table